### PR TITLE
Fix SEO in index.html

### PR DESCRIPTION
**Pull request by DiegoEmilio01**
Basic SEO done. Performance would improve with lower resolution images.
Check console logs, head links cause some and are unkown by me.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
     <meta charset="UTF-8">
     <meta name="keywords" content="">
     <meta name="description" content="">
     <meta name="author" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="shortcut icon" type="image/jpg" href="#" />
+    <link media="screen" rel="shortcut icon" type="image/jpg" href="#" />
     <!-- Inter font -->
-    <link rel="stylesheet" href="https://rsms.me/inter/inter.css" />
+    <link media="screen" rel="preload" as="font" href="https://rsms.me/inter/inter.css" />
```
Comment (nbcl) [inline index.html]: This external resource link loads the desired `.woff` files, but it is still a stylesheet itself, so we should load it as follows.

```html
<link media="screen" rel="stylesheet preload" as="style" href="https://rsms.me/inter/inter.css" />
```

File: index.html [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
-<html>
+<html lang="es">
 
 <head>
     <title>Rheviu</title>
+    <meta name="description" content="¡Opina sobre tus cursos favoritos! ¡Encuentra todos tus ramos en un solo lugar, evaluados por tus compañeros en dificultad, tiempo y mucho más!">
     <meta charset="UTF-8">
     <meta name="keywords" content="">
     <meta name="description" content="">
     <meta name="author" content="">
```
Comment (nbcl) [inline index.html]: Line `9` contain the same metadata element assigned in line `6`, overriding the description.

**Review by nbcl — CHANGES_REQUESTED** [review]
Hey @DiegoEmilio01!

Everything looks good.

I have added two comments about minor fixes that should be considered before merging.

Apart from that, regarding the following reports:

> Performance would improve with lower resolution images.
Check console logs, head links cause some and are unknown by me.

In benefit of the current workflow, please avoid leaving open-ended remarks in pull requests, as this are valid observations that are to be considered by the team and should be reported as individual issues.

**Comment by DiegoEmilio01** [comment]
@nbcl resolved all comments, the pull request should be ready for merge!

**Review by nbcl — APPROVED** [review]

**Pull request merged**